src/App.js

```diff
@@ -1,12 +1,85 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Dynamic list of posts</h1>
-    </div>
-  );
+import PostList from './components/PostList/PostList';
+
+const postsUrl = 'https://jsonplaceholder.typicode.com/posts';
```

Comment:
для подобных констант запись следующая:
```suggestion
const POSTS_API_URL = 'https://jsonplaceholder.typicode.com/posts';
```